### ISSUE TITLE
Nethack with QT build fail on MacOS Catalina.

### ISSUE THREAD

**Issue by PythonB** [NetHack/NetHack#384]
Commands executed:
`./sys/unix/setup.sh sys/unix/hints/macosx10.10`
then
`make fetch-lua`
then
`make`
And make fails with
https://pastebin.com/f4kF9T0V

Qt version is 5.15.0 (installed with Homebrew)
Apple clang version 11.0.0 (clang-1100.0.33.17)
Target: x86_64-apple-darwin19.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin

**Comment by nhmall** [comment]
Try this with the most current NetHack-3.7 branch:

cd sys/unix
sh setup.sh hints/macOS.2020
cd ../..
make fetch-Lua
make clean
make WANT_WIN_QT=1 install

**Comment by PythonB** [comment]
Lua problem was solved by removing folders lib/lua and /lib/lua-5.4.0 and then downloading lua source code from https://www.lua.org/ftp/lua-5.4.0.tar.gz

But now build fails on
`Linking nethack.
clang++ -std=gnu++11  -o nethack monst.o objects.o allmain.o alloc.o apply.o artifact.o attrib.o ball.o bones.o botl.o cmd.o dbridge.o decl.o detect.o dig.o display.o dlb.o do.o do_name.o do_wear.o dog.o dogmove.o dokick.o dothrow.o drawing.o dungeon.o eat.o end.o engrave.o exper.o explode.o extralev.o files.o fountain.o hack.o hacklib.o insight.o invent.o isaac64.o light.o lock.o mail.o makemon.o mapglyph.o mcastu.o mdlib.o mhitm.o mhitu.o minion.o mklev.o mkmap.o mkmaze.o mkobj.o mkroom.o mon.o mondata.o monmove.o mplayer.o mthrowu.o muse.o music.o nhlua.o nhlsel.o nhlobj.o o_init.o objnam.o options.o pager.o pickup.o pline.o polyself.o potion.o pray.o priest.o quest.o questpgr.o read.o rect.o region.o restore.o rip.o rnd.o role.o rumors.o save.o sfstruct.o shk.o shknam.o sit.o sounds.o sp_lev.o spell.o symbols.o sys.o steal.o steed.o teleport.o timeout.o topten.o track.o trap.o u_init.o uhitm.o vault.o vision.o vis_tab.o weapon.o were.o wield.o windows.o wizard.o worm.o worn.o write.o zap.o posixregex.o  ioctl.o unixmain.o unixtty.o unixunix.o unixres.o qt_bind.o qt_click.o qt_clust.o qt_delay.o qt_glyph.o qt_icon.o qt_inv.o qt_key.o qt_line.o qt_main.o qt_map.o qt_menu.o qt_msg.o qt_plsel.o qt_rip.o qt_set.o qt_stat.o qt_str.o qt_streq.o qt_svsel.o qt_win.o qt_xcmd.o qt_yndlg.o tile.o  version.o -F/usr/local/Cellar/qt/5.15.0/lib -framework QtWidgets -F/usr/local/Cellar/qt/5.15.0/lib -framework QtMultimedia -F/usr/local/Cellar/qt/5.15.0/lib -framework QtNetwork -F/usr/local/Cellar/qt/5.15.0/lib -framework QtGui -F/usr/local/Cellar/qt/5.15.0/lib -framework QtCore  ../lib/lua/liblua.a -lm`

with
`Undefined symbols for architecture x86_64:
  "_more", referenced from:
      _docompress_file in files.o
     (maybe you meant: _more_experienced, _set_moreluck , _morehungry )
  "_ospeed", referenced from:
      _gettty in unixtty.o
  "_tty_procs", referenced from:
      _winchoices in windows.o
  "_win_tty_init", referenced from:
      _winchoices in windows.o
ld: symbol(s) not found for architecture x86_64`

Edit: In case someone will encounter same problem in future. BEFORE running make in Nethack directory, I ran make in lib/lua-5.4.0 I don't know if this affects make process or not, but I think it's important to mention.
